### ISSUE TITLE
Forcefield does not allow allies to pass through

### ISSUE THREAD

**Issue by Electroblob77** [Electroblob77/Wizardry#320]
Minecraft version: 1.12.2
Wizardry version: 4.2.5
Environment: LAN game/Server

Issue details: Reported on Discord, forcefield sometimes doesn't allow allies through. Unsure of the conditions to recreate this.

Other mods involved: None

Link to crash report (if applicable): None

**Comment by atmfog** [comment]
Want to bump this bug. Not sure what the proper protocol is on Github, but want to report this bug is still in happening as 2/21/2021 via R.A.D modpack 1.43 from Curseforge. 
I is running on Minecraft 1.12.2 using forge-14.23.5.2854.
This is on hosted server through Akliz, we tried using the forcefield spell with each of individually being on each others allies list and then both being simultaneously. Also tried re-logging after added on both parties but still no luck Still being bumped out of field. Spells such Radiant totem, Healing Aura, Invigorating Presence all functionally appropriately while on allies list.
Not sure what else I could tell you to help you recreate this, but happens 100% of the time since I logged on today.

**Comment by Electroblob77** [comment]
Yeah still on the radar, I think it's part of a wider issue with desyncing of player data (see #409). I know that issue happens but am having trouble reproducing it myself, it's just one of those really difficult issues to debug unfortunately.
